nautobot/utilities/testing/api.py

```diff
@@ -125,14 +128,63 @@ def test_get_object(self):
             obj_perm.users.add(self.user)
             obj_perm.object_types.add(ContentType.objects.get_for_model(self.model))
 
-            # Try GET to permitted object
-            url = self._get_detail_url(instance1)
-            self.assertHttpStatus(self.client.get(url, **self.header), status.HTTP_200_OK)
-
             # Try GET to non-permitted object
             url = self._get_detail_url(instance2)
             self.assertHttpStatus(self.client.get(url, **self.header), status.HTTP_404_NOT_FOUND)
 
+            # Try GET to permitted object
+            url = self._get_detail_url(instance1)
+            response = self.client.get(url, **self.header)
+            self.assertHttpStatus(response, status.HTTP_200_OK)
+            self.assertIsInstance(response.data, dict)
+            # Fields that should be present in *ALL* model serializers:
+            self.assertIn("id", response.data)
+            self.assertEqual(str(response.data["id"]), str(instance1.pk))  # coerce to str to handle both int and uuid
+            self.assertIn("url", response.data)
+            self.assertIn("display", response.data)
+            self.assertIsInstance(response.data["display"], str)
+            # Fields that should be present in appropriate model serializers:
+            if issubclass(self.model, ChangeLoggedModel):
```

Comment:
Maybe instead check `hasattr(self.model, "to_objectchange")` as Gary points out that not all change-logged models inherit from ChangeLoggedModel (why??????)

Reply:
Nope. Only `ChangeLoggedModel` subclasses are guaranteed to have these two fields. Device component template models, for instance, implement `to_objectchange` but don't have these fields.